[2024-04-11T16:37:20.891+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-11T16:37:20.969+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Process_W3_Data.DateTable manual__2024-04-11T16:37:02.536742+00:00 [queued]>
[2024-04-11T16:37:21.007+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Process_W3_Data.DateTable manual__2024-04-11T16:37:02.536742+00:00 [queued]>
[2024-04-11T16:37:21.007+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-04-11T16:37:21.074+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): DateTable> on 2024-04-11 16:37:02.536742+00:00
[2024-04-11T16:37:21.101+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=22668) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-11T16:37:21.116+0000] {standard_task_runner.py:63} INFO - Started process 22673 to run task
[2024-04-11T16:37:21.131+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Process_W3_Data', 'DateTable', 'manual__2024-04-11T16:37:02.536742+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/w3c.py', '--cfg-path', '/tmp/tmp7lpyc80e']
[2024-04-11T16:37:21.138+0000] {standard_task_runner.py:91} INFO - Job 176: Subtask DateTable
[2024-04-11T16:37:21.293+0000] {task_command.py:426} INFO - Running <TaskInstance: Process_W3_Data.DateTable manual__2024-04-11T16:37:02.536742+00:00 [running]> on host 426f0b9f9d71
[2024-04-11T16:37:21.581+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Process_W3_Data' AIRFLOW_CTX_TASK_ID='DateTable' AIRFLOW_CTX_EXECUTION_DATE='2024-04-11T16:37:02.536742+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-11T16:37:02.536742+00:00'
[2024-04-11T16:37:21.587+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-11T16:37:22.099+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-11T16:37:22.101+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-11T16:37:22.118+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Process_W3_Data, task_id=DateTable, execution_date=20240411T163702, start_date=20240411T163720, end_date=20240411T163722
[2024-04-11T16:37:22.192+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-11T16:37:22.237+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-04-11T16:37:22.240+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
